app/tests/src/db/test_bulk_ops.py

```diff
@@ -15,9 +15,14 @@ class Number:
     num: int
 
 
+_next_id = 0
+
+
 def get_random_number_object() -> Number:
+    global _next_id
+    _next_id += 1
     return Number(
-        id=str(random.randint(1000000, 9999999)),
+        id=str(_next_id),
```

Comment:
(not required - just throwing an idea out) You could use Factory boy for building these. While we normally use it for DB models, it also supports creating a dict. Mostly would help with not needing to manage a global counter.
https://github.com/navapbc/simpler-grants-gov/blob/main/api/tests/src/db/models/factories.py#L1269

Not really necessary for just this, but if this was expanded significantly in use I'd recommend it.

Reply:
Oh nice! I'll defer that change for now, thanks for sharing (also, very convenient that the grants.gov project is open source)